src/Text/LLVM/AST.hs

```diff
@@ -1511,11 +1514,14 @@ extendMetadata md stmt = case stmt of
 -- Constant Expressions --------------------------------------------------------
 
 data ConstExpr' lab
-  = ConstGEP Bool (Maybe Word64) Type (Typed (Value' lab)) [Typed (Value' lab)]
+  = ConstGEP [GEPOptionalFlag] (Maybe RangeSpec) Type (Typed (Value' lab)) [Typed (Value' lab)]
```

Comment:
It would be worth documenting the invariant that each unique type of GEP attribute can only appear at most once in this list. (This matters insofar as a `getelementptr` constant would be pretty-printed differently were an attribute to appear duplicated in this list.)

Similarly for the `GEP` data constructor above.

Reply:
Noted as a recommendation, and pretty-printing now nubs these to ensure it outputs correctly despite any duplicates present.